src/test/java/teammates/sqlui/webapi/DeleteFeedbackResponseCommentActionTest.java

```diff
@@ -0,0 +1,721 @@
+        };
+
+        when(mockLogic.getFeedbackResponseComment(typicalFeedbackResponseComment.getId()))
+                .thenReturn(typicalFeedbackResponseComment);
+        when(mockLogic.getInstructorByGoogleId(typicalCourse.getId(), instructorWithoutPrivilege.getGoogleId()))
+                .thenReturn(instructorWithoutPrivilege);
+
+        loginAsInstructor(instructorWithoutPrivilege.getGoogleId());
+
+        verifyCannotAccess(params);
+    }
+
+    @Test
+    void testAccessControl_instructorWithoutRecipientSectionPrivilege_cannotAccessCrossSectionComment() {
+        FeedbackResponseComment typicalFeedbackResponseComment = getTypicalCommentFromTeam();
+
+        Instructor instructorWithoutPrivilege = getTypicalInstructor();
+        instructorWithoutPrivilege.setEmail("instructorWithPrivilege@teammates.tmt");
+        InstructorPrivileges privileges = new InstructorPrivileges();
+        privileges.updatePrivilege("Section A",
+                Const.InstructorPermissions.CAN_MODIFY_SESSION_COMMENT_IN_SECTIONS, true);
+        instructorWithoutPrivilege.setPrivileges(privileges);
+
+        String[] params = new String[] {
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID, typicalFeedbackResponseComment.getId().toString(),
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_RESULT.toString(),
+        };
+
+        when(mockLogic.getFeedbackResponseComment(typicalFeedbackResponseComment.getId()))
+                .thenReturn(typicalFeedbackResponseComment);
+        when(mockLogic.getInstructorByGoogleId(typicalCourse.getId(), instructorWithoutPrivilege.getGoogleId()))
+                .thenReturn(instructorWithoutPrivilege);
+
+        loginAsInstructor(instructorWithoutPrivilege.getGoogleId());
+
+        verifyCannotAccess(params);
+    }
+
+    @Test
+    void testAccessControl_instructorSubmissionPastEndTimeBeforeDeadline_canAccess() {
+        FeedbackSession feedbackSessionPastEndTime = getFeedbackSessionPastEndTime();
+        FeedbackQuestion feedbackQuestion = getTypicalFeedbackQuestionForSession(feedbackSessionPastEndTime);
+        typicalFeedbackResponse = getTypicalFeedbackResponseForQuestion(feedbackQuestion);
+        FeedbackResponseComment typicalFeedbackResponseComment = getTypicalCommentFromInstructorAsParticipant();
+
+        String[] params = new String[] {
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID, typicalFeedbackResponseComment.getId().toString(),
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+        };
+
+        when(mockLogic.getFeedbackResponseComment(typicalFeedbackResponseComment.getId()))
+                .thenReturn(typicalFeedbackResponseComment);
+        when(mockLogic.getInstructorByGoogleId(typicalCourse.getId(), typicalInstructor.getGoogleId()))
+                .thenReturn(typicalInstructor);
+        when(mockLogic.getDeadlineForUser(feedbackSessionPastEndTime, typicalInstructor))
```

Comment:
perhaps you can set this to be a time in the future rather than now - 10 minutes. i think it makes more sense given the test